website/docs/services/fs.mdx

```diff
@@ -0,0 +1,63 @@
+---
+title: Fs
+---
+
+POSIX file system support.
+
+import Docs from '../../../core/src/services/fs/docs.md'
+
+<Docs components={props.components} />
+
+### Via Config
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+<Tabs>
+  <TabItem value="rust" label="Rust" default>
+
+```rust
+use std::sync::Arc;
+
+use anyhow::Result;
+use opendal::services::Azblob;
```

Comment:
We don't need this, instead, we will need `opendal::Scheme`.

Reply:
😨Let me check.

Reply:
Updated.